packages/Webkul/Shop/src/Resources/views/products/view/gallery.blade.php

```diff
@@ -108,96 +108,98 @@ class="min-w-[450px] max-sm:min-w-full w-[490px]"
         </div>
     </script>
 
-    <script type="module">
-        app.component('v-product-gallery', {
-            template: '#v-product-gallery-template',
-    
-            data() {
-                return {
-                    isMediaLoading: true,
-    
-                    media: {
-                        images: @json(product_image()->getGalleryImages($product)),
-
-                        videos: @json(product_video()->getVideos($product)),
-                    },
-    
-                    baseFile: {
-                        type: '',
-
-                        path: ''
-                    },
-    
-                    activeIndex: 0, 
-
-                    containerOffset: 110,
-                }
-            },
-    
-            watch: {
-                'media.images': {
-                    deep: true,
-                    
-                    handler(newImages, oldImages) {
-                        if (JSON.stringify(newImages) !== JSON.stringify(oldImages)) {
-                            this.baseFile.path = newImages[this.activeIndex].large_image_url;
-                        }
-                    },
-                },
-            },
-    
-            mounted() {
-                if (this.media.images.length) {
-                    this.baseFile.type = 'image';
-                    this.baseFile.path = this.media.images[0].large_image_url;
-                } else if (this.media.videos.length) {
-                    this.baseFile.type = 'video';
-                    this.baseFile.path = this.media.videos[0].video_url;
-                }
-            },
+<script type="module">
+    app.component('v-product-gallery', {
+        template: '#v-product-gallery-template',
 
-            computed: {
-                lengthOfMedia() {
-                    return [...this.media.images, ...this.media.videos].length > 5;
-                }
-            },
-    
-            methods: {
-                onMediaLoad() {
-                    this.isMediaLoading = false;
-                },
+        data() {
+            return {
+                isMediaLoading: true,
 
-                change(file, index) {
-                    this.isMediaLoading = true;
+                media: {
+                    images: @json(product_image() - > getGalleryImages($product)),
 
-                    if (file.type == 'videos') {
-                        this.baseFile.type = 'video';
-                        this.baseFile.path = file.video_url;
-                        this.isMediaLoading = false;
-                    } else {
-                        this.baseFile.type = 'image';
-                        this.baseFile.path = file.large_image_url;
-                    }
+                    videos: @json(product_video() - > getVideos($product)),
```

Comment:
Please check this line, there are extra spaces while chaining.

Reply:
Formatted